sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/TextAnalyticsRestClient.cs

```diff
@@ -926,5 +926,81 @@ public Response<SentimentResponse> Sentiment(MultiLanguageBatchInput input, stri
                     throw _clientDiagnostics.CreateRequestFailedException(message.Response);
             }
         }
+
+        internal HttpMessage CreateHealthStatusNextPageRequest(string nextLink, Guid jobId, int? top, int? skip, bool? showStats)
+        {
+            var message = _pipeline.CreateMessage();
+            var request = message.Request;
+            request.Method = RequestMethod.Get;
+            var uri = new RawRequestUriBuilder();
+            uri.AppendRaw(endpoint, false);
+            uri.AppendRaw("/text/analytics/v3.1-preview.3", false);
+            uri.AppendRawNextLink(nextLink, false);
+            request.Uri = uri;
```

Comment:
I want `showStats` properties to be included in the link too.. is there a way to do it?